### ISSUE TITLE
Following hashtags

### ISSUE THREAD

**Issue by meicpalmer** [bluesky-social/social-app#5786]
### Describe the Feature

On Mastodon you can follow a hashtag. I searched and didn't see anyone requesting this feature on here.

Following hashtags would only show posts from that hashtag in the Following feed if the option "Show posts from my feeds" is checked in settings.

There could also be a separate feed called "Followed Hashtags" which shows posts from all of your followed hashtags.

**Comment by Gudine** [comment]
I was gonna say that someone could make a custom feed for that, but it turns out [there already is one](https://bsky.app/profile/mackuba.eu/feed/hashtag), though [it seems to still be in development](https://bsky.app/profile/mackuba.eu/post/3l3vehvzkwsvr), and seems a bit clunky to use.

**Comment by ChrGriffin** [comment]
Dropping in to voice my support for this as a first-party and fully-fledged feature.

**Comment by placoderm** [comment]
I would also appreciate something like this.

**Comment by eigenstil** [comment]
Wow, I can't hardly believe, that this doesn't exist yet as a feature in bluesky. This boggles me.

**Comment by JWDebler** [comment]
I know something similar _did_ exist, because I have added a 'search' column in my blue deck a while ago. But now the option of adding a 'search' column doesn't exist anymore.... 
Please bring it back!

![Image](https://github.com/user-attachments/assets/b66aad56-1d07-403d-a3f1-d43e3461f3a7)

**Comment by Gudine** [comment]
@JWDebler It's worth noting that deck.blue is a third-party app, so its features don't necessarily reflect Bluesky's.

**Comment by Invertex** [comment]
> For this to be more useful, we'd need people to actually USE hashtags a lot more on bsky. Hashtag use is very low because the usability and usefulness of hashtags is also very low in the current client and web UI.

Is there any data on ratio of posts that actually use tags, or are you simply assuming?
For comments/replies, it's understandable people wouldn't bother with hashtags though, so that data would need to be filtered out.

If you're speaking from what you see in general, it should be noted that there is a currently-hidden tags system which some third party services people use to make their posts with put the tags in, such as [https://deck.blue](https://deck.blue)  
Those tags don't show up in the text of the posts, but are in the data of the post and are picked up when you search tags or make feeds for tags.

**Comment by gitcos** [comment]
It's very very obvious that there is extremely low use of hashtags on many major topics and events. I don't know what data there is, but we don't need more data to know there's a significant problem if you want hashtags to be used to anywhere near the level and practical utility they had on Twitter.  It's trivially easy to think of topics or events that would've had hashtags on twitter and find there is no tag being consistently used on bsky.

> Ideally bsky.app would actually expose this on their posts so people don't have to put them into text body....

That is exactly part of what my request https://github.com/bluesky-social/social-app/issues/7693 asks for.  Having hidden tags doesn't help much when the bsky UI doesn't let you add those tags and doesn't let you see when they're on a post so you can't click on them.

**Comment by legowerewolf** [comment]
Yeah, I think Tumblr-style tags are great, sitting outside of the post body.

Although, I suppose given that Bluesky limits post length, you might want to discourage the Tumblrite habit of using the tags to add commentary about a post without it showing up in the reblog chain. But we also don't have reblog chains, so maybe that's not necessary.

**Comment by blueturtleai** [comment]
> It's very very obvious that there is extremely low use of hashtags on many major topics and events.

It‘s a typical hen and egg problem. As long as there is no obvious benefit using hashtags, users don’t use them.

For me following hashtags is a natural and very efficient way discovering content and users I‘m interested in. I hope, that this will be implemented natively soon and the hashtag posts are also displayed in the following feed. In fact it’s the feature I miss the most.

Using feeds isn’t a real alternative, as in many cases no feed exists for a hashtag I‘m interested in. Additionally it’s not very handy, that I have to check every feed separately.

**Comment by gitcos** [comment]
> > It's very very obvious that there is extremely low use of hashtags on many major topics and events.
> 
> It‘s a typical hen and egg problem. As long as there is no obvious benefit using hashtags, users don’t use them.
> 

That's exactly why I posted my feature request, https://github.com/bluesky-social/social-app/issues/7693

That would make hashtags more useful and usable, so then people would also use them more.

Did you vote for my feature request?  Not this one we're commenting on here, which has effectively already been implemented even though it's still open; the one I linked to.

**Comment by blueturtleai** [comment]
> Did you vote for my feature request? Not this one we're commenting on here, which has effectively already been implemented even though it's still open; the one I linked to.

I voted and commented now. Following hashtags has already been implemented?

**Comment by eigenstil** [comment]
> > Did you vote for my feature request? Not this one we're commenting on here, which has effectively already been implemented even though it's still open; the one I linked to.
> 
> I voted and commented now. Following hashtags has already been implemented?

I literally can't believe that following hashtags is available in the android app. I checked it in v1.99.0 which is the latest version to date. And no possibility to follow a hashtag.

**Comment by blueturtleai** [comment]
In the iOS app it doesn’t work either. I hope we get some more details from @gitcos about availability of following hashtags.

**Comment by jlizhiyi** [comment]
Want this too; good replacement for Instagram's similar feature which was removed not too long ago.
